app/routes/companyInterest/components/CompanyInterest.css

```diff
@@ -169,3 +169,9 @@
 .companyInterestListMobile td {
   border: none;
 }
+
+.flag {
+  height: 3em;
+  width: 5em;
```

Comment:
Are you going to start using `em` now too???

Reply:
yes 😢